jobs/mini-e2e.yaml

```diff
@@ -4,6 +4,7 @@
     k8s_version:
       - '1.18'
```

Comment:
can we remove 1.18 as we can test 2 kubernetes versions in CI?

Reply:
need mergify update once PR is merged.

Reply:
yes, but lets first see if things work with this version at all

Reply:
forgot to ask does minikube already supports kubernetes 1.20?

Reply:
no idea, we'll see when the job runs

Reply:
it works with minikube v1.15.1

Reply:
we're on v1.14.1 at the moment, lets see if that works too or if it is required to upgrade